master/buildbot/test/unit/test_stats_service.py

```diff
@@ -154,9 +164,9 @@ def test_influx_service_not_inited(self):
         svc.thd_postStatsValue("test", "test", "test")
         self.assertLogged("Service.*not initialized")
 
-    def test_storage_backend_base(self):
-        # Just initlize and run thd_postStatsValue
-        svc = StatsStorageBase()
+    def test_storage_backend_base_failure_on_init(self):
+        svc = DummyStatsStorageBase()
```

Comment:
Because that will be no different from initializing the parent class itself. Just for testing, I tried doing that and again, pylint raises a post-commit error:

```
(sandbox)[prasoon@jimmypage src]$ git commit -m "Added negative check test"
Validating the following commits:
4be7f823f7f6d01a9641987054947c79cdaa7db4 Added negative check test
** Skipping JavaScript Tests **
** Skipping Python Tests **
-- checking formatting --
-- checking for release notes --
** HEAD~..HEAD does not add release notes **
-- checking import module convention in modified files --
-- running autopep8 --
-- running pep8 --
-- running pyflakes --
-- running pylint --
Warning: option include-ids is deprecated and ignored.
Warning: option symbols is deprecated and ignored.
************* Module buildbot.test.unit.test_stats_service
E:184,47: Abstract class 'DummyStatsStorageBaseNoMethod' with abstract methods instantiated (abstract-class-instantiated)
** pylint failed **
-- not building docs, because it was not changed --

NO GOOD!
** Skipping JavaScript Tests (warning)
** Skipping Python Tests (warning)
** HEAD~..HEAD does not add release notes (warning)
** pylint failed
[todo-pr-2 4be7f82] Added negative check test
 1 file changed, 12 insertions(+)
```